client/src/plus/collections/frequently-viewed.tsx

```diff
@@ -1,31 +1,44 @@
 import { useEffect, useState } from "react";
-import { Doc, DocParent } from "../../../../libs/types/document";
-import { Collection } from "./api";
+import { Doc } from "../../../../libs/types/document";
+import { FrequentlyViewedItem, ItemParent } from "./api";
 
-export interface FrequentlyViewedCollection extends Collection {
-  items: FrequentlyViewedEntry[];
+export interface FrequentlyViewedCollection {
+  name: string;
+  article_count: number;
+  created_at: string;
```

Comment:
nit: I'm not sure this makes a whole lot of sense in the context of FV, and I think we can remove it without any issue